blaze/server/serialization.py

```diff
@@ -34,6 +97,132 @@ def _coerce_str(bytes_or_str):
 )
 
 
+try:
+    import blosc
+    del blosc
+    compress = 'blosc'
+except ImportError:
+    compress = None
+
+
+def _final_base(values):
+    base = values.base
+    while True:
+        try:
+            base = base.base
+        except AttributeError:
```

Comment:
Is there a benefit to stopping on `AttributeError` versus checking `is None`?  The latter seems clearer to me (maybe faster?).